src/test/java/ScreenshotTest.java

```diff
@@ -0,0 +1,42 @@
+import PageFactory.AccountPageFactory;
+import PageFactory.HomePageFactory;
+import config.TestProperties;
+import driver.Driver;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.io.FileHandler;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ScreenshotTest extends BaseTest{
+    private static final String LOGIN = TestProperties.get("login");
+    private static final String PASSWORD = TestProperties.get("password");
+
+    private WebDriver driver;
+
+    @Test
+    public void screenshotTest(){
+        login(LOGIN, PASSWORD);
+
+        AccountPageFactory accountPageFactory = new AccountPageFactory(driver);
+        Assertions.assertTrue(AccountPageFactory.isDisplayed());
+
+        driver = Driver.getDriver();
+        File screenshotFile = ((TakesScreenshot)driver).getScreenshotAs(OutputType.FILE);
+        try{
+            FileHandler.copy(screenshotFile, new File("C:/Users/AnnaMatveenko/OneDrive - Coherent Solutions/SPARTA/Automation/homepage.png"));
```

Comment:
1. let's drop all the screenshots to target/artifacts folder (create it if not exists each time when this code run)
2. let's move this functionality to testng listener (on test failed)
3. let's create full-page screenshot (AShot will help you)
4. and let's add logging instead of having sout (slf4j + logback will be good)